### PR TITLE
Add project description toggle

### LINKED ISSUE #301: Make frontend updates to Round Application page
**Grant Application**
- [x] Add "view project details" toggle
- [x] Remove the $ sign in the description 
- [x] Updating the placeholder text 
- [x] Mark required vs. optional fields 
- [x] New steps for checking if payout wallet address is a multisig (see designs) 
- [x] Check validation for answers on application when user selects "preview application"
- [x] Add multisig questions

Figma: https://www.figma.com/file/obBzCCL8KwoTlVBb5g3ccN/Grant-Hub---Wireframes?node-id=2188%3A10232

Image

Image

Image

Image

Image

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

**Comment by codenamejason** [comment]
Screen Shot 2022-09-15 at 1 10 38 PM

**Comment by codenamejason** [comment]
Screen Shot 2022-09-15 at 2 45 47 PM

File: client/src/components/grants/inputs.tsx [hunk 1]
```diff
@@ -187,8 +187,8 @@ export function Select({
         {label}
       </label>
       {required && (
-        <span className="absolute text-purple-700 inset-y-0 right-0">
-          * Required
```
Comment (DanieleSalatti) [inline client/src/components/grants/inputs.tsx:191]: I've been adding spaces there, are we not supposed to? I can remove them if that's the case

**Review by DanieleSalatti — APPROVED** [review]
LGTM!

File: client/src/components/grants/inputs.tsx (hunk 1 above)
Comment (codenamejason) [reply client/src/components/grants/inputs.tsx:191]: I was trying to follow the Figma and looked like there was no space.

**Pull request merged**